osu.Game.Tests/OnlinePlay/TestCaseCatchUpSyncManager.cs

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using NUnit.Framework;
+using osu.Framework.Bindables;
+using osu.Framework.Testing;
+using osu.Framework.Timing;
+using osu.Game.Screens.OnlinePlay.Multiplayer.Spectate.Sync;
+using osu.Game.Tests.Visual;
+
+namespace osu.Game.Tests.OnlinePlay
+{
+    [HeadlessTest]
+    public class TestCaseCatchUpSyncManager : OsuTestScene
```

Comment:
```suggestion
    [HeadlessTest]
    public class TestSceneCatchUpSyncManager : OsuTestScene
```

+ everything else that is needed (file rename at least)